shimmy/__init__.py

```diff
@@ -45,22 +52,15 @@ def __call__(self, *args: list[Any], **kwargs: Any):
         e,
     )
 
-try:
-    from shimmy.bsuite_compatibility import BSuiteCompatibilityV0
-except ImportError as e:
-    BSuiteCompatibilityV0 = NotInstallClass(
-        "BSuite is not installed, run `pip install 'shimmy[bsuite]'`",
-        e,
-    )
 
 __all__ = [
     "BSuiteCompatibilityV0",
+    "GymV21CompatibilityV0",
```

Comment:
Why have you changed the order?

Reply:
Not sure what this was caused by either, thanks for pointing it out. I think I wanted to group up the dm lab and dm control because alphabetical but gym shouldn’t be before it in that case even.